pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -130,14 +141,23 @@ func (jm *Controller) syncAll() {
 		utilruntime.HandleError(fmt.Errorf("can't list CronJobs: %v", err))
 		return
 	}
-	sjl, ok := sjlTmp.(*batchv1beta1.CronJobList)
-	if !ok {
-		utilruntime.HandleError(fmt.Errorf("expected type *batchv1beta1.CronJobList, got type %T", sjlTmp))
+
+	sjs := make([]batchv1beta1.CronJob, 0)
+	err = meta.EachListItem(sjlTmp, func(object runtime.Object) error {
+		cronJobTmp, ok := object.(*batchv1beta1.CronJob)
```

Comment:
Could you help me understand why object is a pointer to `batchv1beta1.CronJob`?

Reply:
It because in the meta.EachListItem it will change the object in pointer.So we need to change it back from pointer~